Kudu.Core/SiteExtensions/FeedExtensions.cs

```diff
@@ -57,6 +57,23 @@ public static class FeedExtensions
 
             return latestPackage;
         }
+        
+        // TODO refactor last function, after kudu private test all passes
```

Comment:
Not sure I understand why we need this method that looks identical to previous. Can't the previous one just call this one?